### DISCUSSION TITLE
Is there a way to optimize on the size?

### DISCUSSION THREAD

**Discussion by painotpi** [nuxt-community/sentry-module#483] [Q&A]
I'm using this module, the bundle looks pretty big to me,

![image](https://user-images.githubusercontent.com/3349253/209129419-5385c06d-d28b-49a2-9ece-aa3443a700ab.png)

> Parsed size: 98.61 kb
> Gzip size: 26.68 kb

Is this normal or is there a way to optimize the size?

**Comment by rchl** [comment]
There is a way to shave some more by tree shaking default integrations: https://docs.sentry.io/platforms/javascript/guides/vue/configuration/tree-shaking/#tree-shaking-default-integrations

This would also get rid of `localforage` which is pretty big (not included in your screenshot).

That said, it's all SDK's code so not much else we can do besides that.

You can enable `lazy` option which will result in sentry bundle being cached by the browser and effectively only loaded from network on first page load.
